tgui/packages/tgui/interfaces/FTLComputerModular.js

```diff
@@ -0,0 +1,95 @@
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, Section, ProgressBar, Grid, Table } from '../components';
+import { GridColumn } from '../components/Grid';
```

Comment:
```suggestion
import { Box, Button, Section, ProgressBar, Table } from '../components';
```
No need to import unused elements